src/components/Preview.js

```diff
@@ -88,7 +88,7 @@ function Preview({ markup, variant, forwardedRef, dispatch }) {
           ref={refSetter}
           src="/sandbox.html"
           security="restricted"
-          className="w-full h-full"
+          className="absolute w-full h-full"
```

Comment:
I think that instead of adding an `absolute` position here, which would break the element flow, we can remove the wrapping `<div className="flex-auto relative">`, as it doesn't do anything anymore. And only apply the `className="flex-auto"` to the `iframe`. So no `absolute`, `w-full` or `h-full`.